packages/react-components/src/templates/NewsDetailsPage.tsx

```diff
@@ -51,7 +53,7 @@ const NewsDetailsPage: React.FC<NewsDetailsPageProps> = ({
   return (
     <div css={containerStyles}>
       <Card>
-        <Pill>{type}</Pill>
+        {'type' in rest ? <Pill>{rest.type}</Pill> : <Pill>'Tutorial'</Pill>}
```

Comment:
```suggestion
         <Pill>{'type' in rest ? {rest.type} : Tutorial}</Pill>}
```

Reply:
After changing the type of `NewsDetailsPageProps` to the union `Pick<NewsResponse, (...)> | Pick<TutorialsResponse, (...)>`, TS complains if I try to access `rest?.type`. I am using TS narrowing `in` to avoid this error :)

<img width="1095" alt="Screen Shot 2022-10-07 at 13 03 00" src="https://user-images.githubusercontent.com/16595804/194598474-287eff4b-864e-49aa-b362-2355a6b4baca.png">

I will change to this, then

```
<Pill>{'type' in rest ? rest.type : 'Tutorial'}</Pill>
```